provisioning/qa/resources.tf

```diff
@@ -0,0 +1,18 @@
+provider "aws" {
+  region     = "us-east-1"
+}
+
+terraform {
+  # Use s3 to store terraform state
+  backend "s3" {
+    bucket  = "nypl-travis-builds-qa"
```

Comment:
Replace `travis` with `github-actions`. Note that if you do this, you'll need to migrate the state file by running your `init` script for qa and prod with `-migrate-state`

Reply:
Fixed